core/new-gui/src/app/dashboard/component/feature-container/saved-workflow-section/saved-workflow-section.component.spec.ts

```diff
@@ -300,44 +302,44 @@ describe("SavedWorkflowSectionComponent", () => {
     expect(SortedCase).toEqual(["workflow 5", "workflow 4", "workflow 3", "workflow 2", "workflow 1"]);
   });
 
-  it("searchNoInput", () => {
+  it("searchNoInput", async () => {
```

Comment:
provide brief descriptions for each test case. one sentence is enough.

Reply:
comments are added now.